crates/sui-sdk-types/src/types/object.rs

```diff
@@ -1,5 +1,7 @@
 use std::collections::BTreeMap;
 
+#[cfg(all(feature = "serde", feature = "hash"))]
+use super::unresolved::Input;
```

Comment:
Give this is used in only a single place, lets inline the import to avoid needing to handle the conditional compilation in two places.